cmd/tools/cli/plugins/authorization-plugin.go

```diff
@@ -0,0 +1,62 @@
+// The MIT License
```

Comment:
Let's move this to `authorization/main.go`. We supposed to have more plugins, right? And plugin might have more than 1 file in it.